SPECS/iptables/iptables.conf

```diff
@@ -0,0 +1,11 @@
+ip_tables
+iptable_filter
+iptable_mangle
+iptable_nat
+iptable_security
+ip6_tables
+ip6table_filter
+ip6table_mangle
+ip6table_nat
+ebt_ip
+nf_nat
```

Comment:
should we add these modules as well:
```
ipt_ah 
ipt_ECN 
ipt_REJECT 
ipt_rpfilter 
ipt_SYNPROXY
```

Rationale: When i did a repro locally, I could see these modules also

Reply:
Were these enabled by default with the previous iptables package or only after the repro? If they weren't loaded before the repro, I don't think we'd want to add them to the conf file